src/icons.qrc

```diff
@@ -251,7 +251,6 @@
         <file>icons/flags/zw.svg</file>
         <file>icons/L.gif</file>
         <file>icons/loading.png</file>
-        <file>icons/qbittorrent.png</file>
```

Comment:
The webUI login screen is missing the qbt logo, I guess is due to this line.

@glassez 
Since you have #7610, maybe you can fix this in your PR, or let me know to open a PR for this.